CMakeLists.txt

```
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(ExaMiniMD LANGUAGES CXX)
+
+find_package(Kokkos REQUIRED)
```

```suggestion
find_package(Kokkos 3.0 REQUIRED)
```
Before 3.0 there is no good cmake support.

---

aba28eca8aac1f916b1eaecf6d2c45f3fbd2fed2